tailwind.config.js

```diff
@@ -0,0 +1,169 @@
+      4: theme("colors.border.4"),
+      5: theme("colors.border.5"),
+    }),
+    borderWidth: {
+      0: "0",
+      1: "1px",
+      3: "3px"
+    },
+    borderRadius: {
+      none: '0',
+      1: "var(--calcite-border-radius)",
+      2: "30px",
+      3: "50px",
+      half: '50%',
+      full: '100%',
+    },
+    boxShadow: {
+      1: {
+        default: "0 0 8px 0 rgba(0, 0, 0, 0.08)",
+        hover: "0 0 16px 0 rgba(0, 0, 0, 0.08)",
+        press: "0 0 8px 0 rgba(0, 0, 0, 0.16)"
+      },
+      2: {
+        default: "0 0 16px 0 rgba(0, 0, 0, 0.16)",
+        hover: "0 0 32px 0 rgba(0, 0, 0, 0.16)",
+        press: "0 0 16px 0 rgba(0, 0, 0, 0.32)",
+      }
+    },
+    colors: {
+      blue: {
+        1: "var(--calcite-ui-blue-1)",
+        2: "var(--calcite-ui-blue-2)",
+        3: "var(--calcite-ui-blue-3)",
+      },
+      green: {
+        1: "var(--calcite-ui-green-1)",
+        2: "var(--calcite-ui-green-2)",
+        3: "var(--calcite-ui-green-3)",
+      },
+      yellow: {
+        1: "var(--calcite-ui-yellow-1)",
+        2: "var(--calcite-ui-yellow-2)",
+        3: "var(--calcite-ui-yellow-3)",
+      },
+      red: {
+        1: "var(--calcite-ui-red-1)",
+        2: "var(--calcite-ui-red-2)",
+        3: "var(--calcite-ui-red-3)",
+      },
+      background: "var(--calcite-ui-background)",
+      foreground: {
+        1: "var(--calcite-ui-foreground-1)",
+        2: "var(--calcite-ui-foreground-2)",
+        3: "var(--calcite-ui-foreground-3)",
+      },
+      text: {
+        1: "var(--calcite-ui-text-1)",
+        2: "var(--calcite-ui-text-2)",
+        3: "var(--calcite-ui-text-3)",
+      },
+      border: {
+        1: "var(--calcite-ui-border-1)",
+        2: "var(--calcite-ui-border-2)",
+        3: "var(--calcite-ui-border-3)",
+        4: "var(--calcite-ui-border-4)",
+        5: "var(--calcite-ui-border-5)",
+      }
+    },
+    fontFamily: {
+      sans: ["Avenir Next", "Avenir", "Helvetica Neue", "sans-serif"],
+      mono: ["Consolas", "Andale Mono", "Lucida Console", "Monaco", "monospace"],
+    },
+    fontSize: {
+      "-3": "0.625rem", // 10px
+      "-2": "0.75rem",  // 12px
+      "-1": "0.875rem", // 14px
+      0: "1rem",        // 16px
+      1: "1.125rem",    // 18px
+      2: "1.25rem",     // 20px
+      3: "1.625rem",    // 26px
+      4: "2rem",        // 32px
+      5: "2.5rem",      // 40px
+      6: "3rem",        // 48px
+      7: "3.5rem",      // 56px
+      8: "4rem",        // 64px
+    },
+    fontWeight: {
+      light: 300,
+      normal: 400,
+      medium: 500,
+      bold: 700
+    },
+    minWidth: theme => ({
+      ...theme("spacing")
+    }),
+    minHeight: theme => ({
+      ...theme("spacing")
+    }),
+    opacity: {
+      0: '0',
+      20: '0.2',
+      30: '0.3',
+      40: '0.4',
+      80: '0.8',
+      100: '1'
+    },
+    screens: {
+      's': '480px',
+      'm': '860px',
+      'l': '1024px',
+      'xl': '1440px'
```

Comment:
It might be worth adding some doc around our areas of customization as to how we landed on our values. I actually had to do [some](https://github.com/tailwindlabs/tailwindcss/pull/683) [digging](https://github.com/tailwindlabs/tailwindcss/pull/759) to find the rationale behind the default breakpoints).

At some point, we may want to revisit the `m` breakpoint of `860px` (`53.75rem`). Looking at the defaults, it looks like they are all even integers and share `1rem` (16px) as a factor. This might allow for things to align more naturally.

Reply:
Updated `m` to be divisible by 16 as well, because that feels nice and cohesive. In all honestly I'm not sure where these values came from, probably bootstrap?